integration/app/store/store.config.ts

```diff
@@ -0,0 +1,22 @@
+import { NgxsConfig } from '@ngxs/store';
+import { NgxsDevtoolsOptions } from '@ngxs/devtools-plugin';
+import { NgxsLoggerPluginOptions } from '@ngxs/logger-plugin';
+
+import { environment as env } from '@integration/env/environment';
+import { TodosState } from '@integration/store/todos/todos.state';
+import { TodoState } from '@integration/store/todos/todo/todo.state';
+
+export const STATES_MODULES = [TodosState, TodoState];
+
+export const OPTIONS_CONFIG: Partial<NgxsConfig> = {
+  developmentMode: !env.production
```

Comment:
This is not related to this PR, but this line and line 16 got no meaning,
because the `serve:integration` script uses the `--prod` flag.

Reply:
I think it is good to show an example of what the user should do in their app.

Reply:
I agree, but I think we should remove the --prod flag in the scripts.